contracts/cw-proposal-multiple/src/msg.rs

```diff
@@ -158,6 +158,13 @@ pub enum QueryMsg {
     },
     ProposalHooks {},
     VoteHooks {},
+    FilterListProposals {
```

Comment:
Would be great to add a little docstring to this method describing what each field does. :)